docs/tools/config_system.md

```diff
@@ -88,90 +213,33 @@ In this JSON file:
 
 - `name` is the name of the library. **This is a required field.**
 - `config` defines the configuration parameters of the library, as explained in the section about [defining configuration parameters](#defining-configuration-parameters).
-- `macros` is a list of extra macros that will be defined when compiling a project that includes this library. A macro can be defined without a value (such as `MYMOD_MACRO1` above) or with a value (such as `MYMOD_MACRO2` above).
+- `macros` is a list of extra macros that will be defined when compiling a application that includes this library. A macro can be defined without a value (such as `MYMOD_MACRO1` above) or with a value (such as `MYMOD_MACRO2` above).
 - `target_overrides` is a dictionary with target-specific values for the configuration parameters.
 
-`target_overrides` is used to override the values of the parameters depending on the current compilation target. The keys in `target_overrides` are matched against toolchain *labels*. (You can find a description of Mbed targets in our documentation about [adding and configuring targets](/docs/development/tools/adding-and-configuring-targets.html).) If a key inside `target_overrides` matches one of the target labels, the parameter values are changed according to the value of the key. In the example above:
+All configuration parameters defined in `mylib` have a `mylib.` prefix. In a `mbed_app.json`, `buffer_size` is accessible using the name `mylib.buffer_size`.
+`target_overrides` is used to override the values of the parameters depending on the current compilation target. The keys in `target_overrides` are matched against target labels. (You can find a description of Mbed targets in our documentation about [adding and configuring targets](/docs/development/tools/adding-and-configuring-targets.html).) If a key inside `target_overrides` matches one of the target labels, the parameter values are changed according to the value of the key. 
```

Comment:
Link is broken when using "view" here. Does it work in the main repo or on the website?

Reply:
@AnotherButler Correct me if I'm wrong, but I think this works on the website.